generator/sbpg/targets/resources/sbp_construct_template.py.j2

```diff
@@ -55,33 +58,42 @@ class ((( m.identifier )))(object):
                '((( f.identifier )))',
                ((*- endfor *))
               ]
+  __zips__ = [((* for f in m.fields *))
```

Comment:
I don't really like using an __ variables because it makes me think this is some builtin python object on the class rather than something we are defining ourselves:  Can we rename this object to describe what it really is? Could something like "msg_dict" be appropriate, or _fields ?